recipes/capytaine/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "1.0" %}
+
+package:
+  name: capytaine
+  version: {{ version }}
+
+source:
+  url: https://github.com/mancellin/capytaine/archive/v{{version}}.tar.gz
+  sha256: 072de801b50b4052277b5eb3d634734a11079b69789e5fb3bccb7363d199f637
+
+build:
+  skip: True  # [py<36]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+  host:
+    - python >= 3.6
+    - pip
+    - setuptools
+    - numpy
+    - llvm-openmp  # [osx]
+  run:
+    - python >= 3.6
+    - attrs
+    - {{ pin_compatible('numpy') }}
+    - pandas
+    - matplotlib
+    - scipy
+    - vtk
+    - xarray
+    - llvm-openmp  # [osx]
+
+test:
+  imports:
+    - capytaine
```

Comment:
Please, check if this import will bring binary extensions. If not, it would be useful to add more imports in order to check compilation.

Reply:
Yes, the import raises an error when the binary extension is missing.